src/App.tsx

```diff
@@ -52,7 +54,29 @@ function App() {
         />
         <Route
           path="/wishlist"
-          element={<WishList />}
+          element={
+            isAuth ? (
+              <WishList />
+            ) : (
+              <Navigate
+                to="/"
+                replace
+              />
+            )
+          }
+        />
+        <Route
+          path="/user"
+          element={
+            isAuth ? (
+              <User />
+            ) : (
+              <Navigate
+                to="/"
+                replace
+              />
+            )
+          }
```

Comment:
Try to find another way
If you have 10 auth routes it doesn't make sense to describe them like here